staging/src/k8s.io/api/README.md

```diff
@@ -6,6 +6,29 @@ Schema of the external API types that are served by the Kubernetes API server.
 
 This library is the canonical location of the Kubernetes API definition. Most likely interaction with this repository is as a dependency of client-go.
 
+It is published separately to avoid diamond dependency problems for users who
+depend on more than one of k8s.io/kubernetes/client-go,
+k8s.io/kubernetes/apimachinery, k8s.io/kubernetes/apiserver...
+
+## Recommended Use
+
+We recommend using the go types in this repo. In most cases, you may serialize
```

Comment:
I'm not aware of any cases where you cannot serialize directly to JSON... were there some you were thinking of?

Reply:
Nope, just thought of the places where we have a second pass (e.g. RawExtension) and added weasel words.